charmcraft.yaml

```diff
@@ -98,15 +98,18 @@ config:
     root-ca-validity:
       type: int
       default: 365
-      description: RootCA certificate validity (in days).
+      description: RootCA certificate validity (in days), should longer than twice of certificate-validity.
     certificate-validity:
       type: int
       default: 365
-      description: Certificate validity (in days).
+      description: Certificate validity (in days) must be shorter than half the root-ca-validity.
```

Comment:
The default values don't match with the description

Reply:
✅